CONTRIBUTING.md

```diff
@@ -1,14 +1,27 @@
+Issue report guidelines
```

Comment:
Shouldn't we create an [issue template](https://github.com/blog/2111-issue-and-pull-request-templates)?